content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -82,43 +70,28 @@ node {
 }
 ----
 
-Not all Pipelines will have these same three stages, but it is a good starting
-point to define them for most projects. The sections below will demonstrate the
-creation and execution of a simple Pipeline in a test installation of Jenkins.
+Not all Pipelines will have these same three stages, but it is a good starting point to define them for most projects.
+The sections below will demonstrate the creation and execution of a simple Pipeline in a test installation of Jenkins.
 
 [NOTE]
 ====
-It is assumed that there is already a source control repository set up for
-the project and a Pipeline has been defined in Jenkins following
-<<getting-started#defining-a-pipeline-in-scm, these instructions>>.
+It is assumed that there is already a source control repository set up for the project and a Pipeline has been defined in Jenkins following <<getting-started#defining-a-pipeline-in-scm, these instructions>>.
 ====
 
-Using a text editor, ideally one which supports
-link:http://groovy-lang.org[Groovy]
-syntax highlighting, create a new `Jenkinsfile` in the root directory of the
-project.
+Using a text editor, ideally one which supports link:http://groovy-lang.org[Groovy] syntax highlighting, create a new `Jenkinsfile` in the root directory of the project.
 
 
-The Declarative Pipeline example above contains the minimum necessary structure
-to implement a continuous delivery pipeline. The <<syntax#agent, agent
-directive>>, which is required, instructs Jenkins to allocate an executor and
-workspace for the Pipeline. Without an `agent` directive, not only is the
-Declarative Pipeline not valid, it would not be capable of doing any work! By
-default the `agent` directive ensures that the source repository is checked out
-and made available for steps in the subsequent stages.
+The Declarative Pipeline example above contains the minimum necessary structure to implement a continuous delivery pipeline.
+The <<syntax#agent, agent directive>>, which is required, instructs Jenkins to allocate an executor and workspace for the Pipeline.
+Without an `agent` directive, not only is the Declarative Pipeline not valid, it would not be capable of doing any work!
+By default the `agent` directive ensures that the source repository is checked out and made available for steps in the subsequent stages.
 
-The <<syntax#stages, stages directive>>, and <<syntax#steps, steps directives>>
-are also required for a valid Declarative Pipeline as they instruct Jenkins
-what to execute and in which stage it should be executed.
+The <<syntax#stages, stages directive>>, and <<syntax#steps, steps directives>> are also required for a valid Declarative Pipeline as they instruct Jenkins what to execute and in which stage it should be executed.
```

```suggestion
The <<syntax#stages, stages directive>> and <<syntax#steps, steps directives>> are also required for a valid Declarative Pipeline as they instruct Jenkins what to execute and in which stage it should be executed.
```
Suggestion to remove unneeded comma.